float64 avg
float64 var
float64 min
float64 max
